tests/testthat/test-object-package.R

```diff
@@ -0,0 +1,34 @@
+context("Object: Package")
+
+test_that("author with one given name ", {
+  expect_equal(author_desc(list(given="Given",
+                                family="Family",
+                                role=c("aut", "cre"),
+                                email="given.family@email.tld",
+                                comment=NULL)),
+               "Given Family \\email{given.family@email.tld}"
+  )
+})
+
+test_that("author with more than one given name ", {
+  expect_equal(author_desc(list(given=c("First", "Second"),
+                                family="Family",
+                                role=c("aut", "cre"),
+                                email="first.second.family@email.tld",
+                                comment=NULL)),
+               "First Second Family \\email{first.second.family@email.tld}"
+  )
+})
+
+test_that("Authors@R with more than one given name ", {
```

Comment:
Since you modify only `author_desc()`, I wouldn't bother adding tests for other functions.

Reply:
Removed